.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,17 @@
+name: Build & Test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build with Gradle
+        run: ./gradlew clean build
```

Comment:
why not this:
```suggestion
        run: ./gradlew clean test
```

Reply:
Well, build also runs test as part of it... but yeah, just running test will be faster, and it'll compile all the code as well. I'll change it to test.